src/FunderMaps.Core/Interfaces/IBlobStorageService.cs

```diff
@@ -49,14 +49,21 @@ public interface IBlobStorageService
 
         // FUTURE: Refactor
         /// <summary>
-        ///     Stores a file in a Digital Ocean Space.
+        ///     Stores a directory in a Digital Ocean Space.
```

Comment:
Amazon S3

Reply:
The fact that this driver will communicate to another service URL (Digital Ocean) is a config matter. There should not be any mention of Digital Ocean in the code